cpg-language-python/src/test/kotlin/de/fraunhofer/aisec/cpg/frontends/python/PythonFrontendTest.kt

```diff
@@ -1923,4 +1923,22 @@ class PythonFrontendTest : BaseTest() {
         assertNotNull(myClass)
         assertNotNull(myClass.ancestors)
     }
+
+    @Test
+    fun testNestedReplace() {
```

Comment:
I have no idea what is the purpose of this test. We expect to infer functions in the translation unit where it is missing?

Reply:
This was needed because of an intricate bug that occured when we need to infer something based on a node that is an argument of a call expression that was replaced :D